src/categorizer.py

```
@@ -26,146 +28,668 @@ def __init__(
+            "max_workers": self.max_workers,
+            "worker_type": "ThreadPoolExecutor",
+            "monitoring": True,
+            "graceful_shutdown": True,
+            "performance_metrics": {
+                "total_workers": self.max_workers,
+                "cpu_count": os.cpu_count(),
+                "recommended_workers": min(os.cpu_count(), 4)
+            }
+        }
+        
+        print(f"🚀 Configurando executor paralelo:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)